sonar-plugin-api/src/main/java/org/sonar/api/issue/batch/IssueFilter.java

```diff
@@ -26,10 +26,12 @@
 /**
  * <p>An issue filter is an object that allows filtering of {@link Issue}s on batch side, preventing them from being persisted.</p>
  * @since 4.0
+ * @deprecated since 5.3. Use {@link org.sonar.api.issue.filter.IssueFilter} instead.
  *
  */
 @BatchSide
 @ExtensionPoint
+@Deprecated
 public interface IssueFilter {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)